packages/core/src/components/BaseRadio/BaseRadio.styles.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Radio as MuiRadio,
+  radioClasses as MuiRadioClasses,
+} from "@mui/material";
+import styled from "@emotion/styled";
+import { theme } from "@hitachivantara/uikit-styles";
+import { outlineStyles } from "utils";
+import { transientOptions } from "utils/transientOptions";
+
+export const StyledRadio = styled(
+  MuiRadio,
+  transientOptions
+)(({ $focusVisible }: { $focusVisible: boolean }) => ({
+  padding: theme.baseRadio.padding,
+  cursor: "pointer",
+  "&:hover": {
+    backgroundColor: theme.baseRadio.hoverColor,
+  },
+  borderRadius: 0,
+  "&:disabled": {
```

Comment:
For some reason, this styling is not being applied. I think the root element doesn't have the `disabled` attribute for this selector to be applied. But I think passing this inside `MuiRadioClasses.disabled` should work.

Reply:
You are right, fixed it in this next push